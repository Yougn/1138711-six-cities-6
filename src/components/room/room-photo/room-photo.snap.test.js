import React from 'react';
import {render} from '@testing-library/react';
import RoomPhoto from './room-photo';


test(`Should RoomPhoto render correctly`, () => {
  const {container} = render(<RoomPhoto />);
  expect(container).toMatchSnapshot();
});
