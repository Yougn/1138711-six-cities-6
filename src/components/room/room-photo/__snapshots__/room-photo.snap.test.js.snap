// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should RoomPhoto render correctly 1`] = `
<div>
  <div
    class="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      class="property__image"
    />
  </div>
</div>
`;
